az-core/src/main/java/azkaban/Constants.java

```diff
@@ -130,6 +130,12 @@ public class Constants {
   // Used by UserManager for password validation (to tell apart real passwords from auth codes).
   // Empirically, passwords are shorter than this, and ACs are longer:
   public static final int OAUTH_MIN_AUTHCODE_LENGTH = 80;
+  // Hadoop Security Manager retry policy constants
+  public static final int HSM_MAX_RETRY_ATTEMPTS = 5;
```

Comment:
Can you please move these constants to HSM2 code it self? They are only used in that code only. Having them declared within the class should be good enough.